### PR TITLE
Fix get mac address table

### PR DESCRIPTION
**Pull request by dmcken**
The Ethernet switch section has been re-absorbed back into the generic bridge implementation (controlled by the hardware offload tag).

Mikrotik has a concept of untagged similar to Juniper in the case where the vlan ID is not set it is set to 0.

### CODE REVIEW

File: napalm_ros/ros.py [hunk 1]
```diff
@@ -87,18 +87,20 @@ def get_arp_table(self, vrf=""):
...
         table = list()
-        for entry in self.api('/interface/ethernet/switch/unicast-fdb/print'):
+        for entry in self.api('/interface/bridge/host/print'):
             table.append(
                 dict(
                     mac=entry['mac-address'],
-                    interface=entry['port'],
-                    vlan=entry['vlan-id'],
+                    interface=entry['interface'],
+                    # The vid is not consistently set in the API
```
Comment (luqasz) [inline napalm_ros/ros.py:95]: What do you mean ?

File: napalm_ros/ros.py [hunk 2]
```diff
@@ -87,18 +87,20 @@ def get_arp_table(self, vrf=""):
 
     def get_mac_address_table(self):
         table = list()
-        for entry in self.api('/interface/ethernet/switch/unicast-fdb/print'):
```
Comment (luqasz) [inline napalm_ros/ros.py:90]: Both `switch` and `bridge` needs to be handeled in this method. Backwarards compatibility.

File: napalm_ros/ros.py (hunk 1 above)
Comment (dmcken) [reply napalm_ros/ros.py]: The vlan entry is not always coming down and hence 'vlan-id' doesn't exist in the entry dictionary.

File: napalm_ros/ros.py (hunk 2 above)
Comment (dmcken) [reply napalm_ros/ros.py:90]: Hmm, ok. I can re-add it. It was only used with one hardware series so I'm honestly surprised this was the method used, the entry doesn't exist on any other devices as far as I can see, checked CHRs, CRS3XX series, CCR).

https://wiki.mikrotik.com/wiki/Manual:CRS1xx/2xx_series_switches

This is vs the bridge implementation which is supported on all Mikrotik devices.

Should I return a combined table with both sets of data?

**Comment by dmcken** [comment]
I'd like to close this till I get the unit tests fixed.

**Pull request closed without merging**